tasks/40_find-friend/find-friend.js

```diff
@@ -0,0 +1,19 @@
+function findFriend(arr) {
+  const redMask = 'red';
+  const blueMask = 'blue';
+  let counter = 0;
+  arr.forEach((curr, i) => {
+    if (
+      (curr === redMask && arr[i + 1] === blueMask && arr[i + 2] === blueMask)
```

Comment:
forEach is looping over the same values as regular for. My strong suggestion is to go starting from 2 ending up to length-2. But need to think about corner cases

Reply:
I fixed it, but it seems very big :)